mobile/src/main/java/org/hawkular/client/android/backend/model/Trigger.java

```diff
@@ -74,13 +82,15 @@ public Trigger createFromParcel(Parcel parcel) {
     private Trigger(Parcel parcel) {
         this.id = parcel.readString();
         this.description = parcel.readString();
+        this.enabled = (boolean) parcel.readValue(null);
```

Comment:
As from json `true` and `false will be received as Strings. You can have a look [example](at https://github.com/hawkular/hawkular-android-client/blob/master/mobile/src/main/java/org/hawkular/client/android/backend/model/MetricAvailabilityBucket.java#L42) for how you should save and receive them